test/fuzz/BridgeCommitteeFuzzTest.t.sol

```diff
@@ -0,0 +1,217 @@
+// SPDX-License-Identifier: MIT
+pragma solidity ^0.8.20;
+
+import "./BridgeBaseFuzzTest.t.sol";
+import "../../contracts/BridgeCommittee.sol";
+import "../../contracts/utils/BridgeMessage.sol";
+
+contract BridgeCommitteeFuzzTest is BridgeBaseFuzzTest {
+    BridgeCommittee public bridgeCommittee;
+
+    address committeeMemeberAddressA;
```

Comment:
might want to create a helper function that creates a new bridge committee with fuzzed states.